### PR TITLE
Send email for rejected donation form

### PR DESCRIPTION
**Pull request by kevin-wangg**
- Email is sent to donor when a donation form is rejected
- Also emails are now sent from TPC's no reply email

### CODE REVIEW

File: client/src/components/utils/donationForm.ts [hunk 1]
```diff
@@ -53,3 +53,11 @@ export const getContactName = (contact?: DonationFormContact): string | null =>
     }
     return lastName;
 };
+
+export const getContactEmail = (contact?: DonationFormContact): string | null => {
+    const email = contact?.email ?? "";
+    if (contact == null || email.length == 0) {
```
Comment (meganniu) [inline client/src/components/utils/donationForm.ts]: nit: use `email.length === 0`

File: client/src/components/molecules/UnmatchedDonationFormsTable.tsx [hunk 2]
```diff
@@ -136,6 +152,13 @@ const UnmatchedDonationFormsTable: FunctionComponent<Props> = (props: Props) =>
         setDonationForms(donationForms.filter((donationForm) => donationForm._id !== donationFormId));
     };
 
+    const onRejectDonationForm = (donationForm: DonationForm) => {
+        console.log(donationForm._id);
```
Comment (meganniu) [inline client/src/components/molecules/UnmatchedDonationFormsTable.tsx]: stray console log

**Review by meganniu — APPROVED** [review]
Beautiful :')
Thanks @kevinwang2040 
Just make sure to push the new .env to secret manager

File: server/api/utils/email.ts [hunk 3]
```diff
@@ -20,13 +20,38 @@ const getItemAgeDescription = (age: number): string => {
...
+        htmlString += `Item Description: ${item.description}`;
+    }
+    htmlString +=
+        "</p><p>Please contact us with any questions at info@thepregnancycentre.ca and please consider donating other items in the future.</p>The Pregnancy Centre</body>";
+
+    await transporter.sendMail({
+        from: '"no-reply " <noreply@pregnancycentre.ca>', // sender address
+        to: email, // list of receivers
+        subject: "TPC's Donation Review", // Subject line
+        text: "Your donation form has been approved", // plain text body
```
Comment (Puepis) [inline server/api/utils/email.ts]: i think this should say "Your donation form has been rejected"

**Review by Puepis — CHANGES_REQUESTED** [review]
just a wording change in the rejection email text

File: server/api/utils/email.ts (hunk 3 above)
Comment (kevin-wangg) [reply server/api/utils/email.ts]: Fixed.

File: client/src/components/molecules/UnmatchedDonationFormsTable.tsx (hunk 2 above)
Comment (kevin-wangg) [reply client/src/components/molecules/UnmatchedDonationFormsTable.tsx]: Removed.

File: client/src/components/utils/donationForm.ts (hunk 1 above)
Comment (kevin-wangg) [reply client/src/components/utils/donationForm.ts]: Updated.

**Review by Puepis — APPROVED** [review]
LGTM!

**Pull request merged**